cellarium/cas/client.py

```diff
@@ -380,6 +407,12 @@ def __prepare_input_for_sharded_request(
         self._print(f"Cellarium CAS (Model ID: {cas_model_name})")
         self._print(f"Total number of input cells: {len(adata)}")
 
+        try:
+            self.__validate_cells_under_quota(cell_count=len(adata))
+        except exceptions.QuotaExceededError as e:
+            self._print(str(e))
+            sys.exit(1)
```

Comment:
I thought that too, but wasn't sure what the right way to do it was.  Obviously, we don't want it to keep processing if it exceeds quota.  I could raise the error, but I guess I was thinking it would be best to just print the error message in a user readable way and not also print a stack trace.  Do you know what like the idiomatic Python way to do that would be?

Reply:
I just wonder if someone is running this in a python context, won't that kill the program?  I guess I'm wondering if we need anything more than just raising the error?

Reply:
Oh yeah, that's a good point.  I'm probably overcomplicating things.  I'll change it to just raise the error